BaseStation/MainWindow/UserInterface.csproj

```diff
@@ -39,6 +39,42 @@
   <PropertyGroup>
     <StartupObject />
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
```

Comment:
Do you know why this was modified?

Reply:
not sure. maybe similar to the comment lower